nltk/collocations.py

```diff
@@ -122,6 +123,22 @@ def above_score(self, score_fn, min_score):
                 break
 
 
+
+def _build_new_documents(documents, window_size, pad_left=False, pad_right=False, pad_symbol=None):
```

Comment:
In case someone depends on `None` being an acceptable token (is this reasonable?), you could use `PAD_SYMBOL = object()` to create a singleton and always use that symbol here.